UI/app_update.tsx

```diff
@@ -423,7 +423,7 @@ export async function* UI_Interaction_Update(args: {
                 },
                 {
                     title: "Relays",
-                    fields: await app.database.getRelayRecord(nostrEvent.id),
+                    fields: Array.from(app.database.getRelayRecord(nostrEvent.id)),
```

Comment:
Maybe `fields` should accept `iterable` so that you don't have to change the set to an array. Not a big deal.